src/steps/ms-defender/machine/index.ts

```diff
@@ -90,8 +95,7 @@ export async function fetchEndpoints({
   );
 
   if (numFailedEndpointDetails > 0) {
-    // Should this be a user notification instead of an error?
-    throw new IntegrationError({
+    logger.warn({
```

Comment:
We want to only log this internally and not to the user?

Reply:
Exactly, this POV is getting a step failed for 4 machines out of 19.990 so I would like not to generate an integration error out of it.